umh-core/pkg/fsm/benthos/actions.go

```diff
@@ -299,7 +299,7 @@ func (b *BenthosInstance) IsBenthosS6Running() (bool, string) {
 //	reason – empty when ok is true; otherwise a short operator-friendly
 //	         explanation.
 func (b *BenthosInstance) IsBenthosS6Stopped() (bool, string) {
-	if b.ObservedState.ServiceInfo.S6FSMState == s6fsm.OperationalStateStopped {
+	if b.ObservedState.ServiceInfo.S6FSMState == s6fsm.OperationalStateStopped || b.ObservedState.ServiceInfo.S6FSMState == "" {
```

Comment:
Can you add a code comment explaining why this is necessary? ideally put in the whole context from the Linear ticket explaining this hotfix and what it fixes